parboiled-core/src/test/java/CoreTest.java

```diff
@@ -5,13 +5,11 @@
 import java.io.File;
 import java.io.IOException;
 
-import static scala.collection.JavaConversions.collectionAsScalaIterable;
-
 @SuppressWarnings("unchecked")
 public class CoreTest extends TestNGWrapperSuite {
 
     public CoreTest() throws IOException {
-        super(collectionAsScalaIterable(ImmutableList.of(getSuiteFileName())).toList());
+        super(new org.parboiled.JavaCollectionWrapper(ImmutableList.of(getSuiteFileName())).toList());
```

Comment:
You should still be able to use `JavaConverters`. Why do you prefer writing your own wrapper?

Reply:
This is a Java source code. We should explicit method call (can't use `.asScala`). Scala 2.10, 2.11 does not have same name method.

```
Welcome to Scala 2.13.0-M4 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_172).
Type in expressions for evaluation. Or try :help.

scala> scala.collection.JavaConverters.collectionAsScalaIterable(java.util.Collections.emptyList[Int])
res0: Iterable[Int] = Wrappers.JCollectionWrapper()
```

```
Welcome to Scala version 2.10.7 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_172).
Type in expressions to have them evaluated.
Type :help for more information.

scala> scala.collection.JavaConverters.collectionAsScalaIterable(java.util.Collections.emptyList[Int])
<console>:8: error: value collectionAsScalaIterable is not a member of object scala.collection.JavaConverters
              scala.collection.JavaConverters.collectionAsScalaIterable(java.util.Collections.emptyList[Int])
                                              ^
```